azclishell/color_styles.py

```diff
@@ -0,0 +1,253 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import platform
+
+from prompt_toolkit.styles import style_from_dict
+from pygments.token import Token
+
+
+# pylint: disable=R0913
```

Comment:
Use the word form instead of the code.
Someone looking at this would have no idea what R0913 is.
https://docs.pylint.org/en/1.6.0/features.html#id7